mscore/palette/palettewidget.cpp

```diff
@@ -149,8 +149,7 @@ void PaletteWidget::activateSearchBox()
 
 void PaletteWidget::applyCurrentPaletteElement()
       {
-      const bool invoked = QMetaObject::invokeMethod(rootObject(), "applyCurrentPaletteElement");
-      Q_ASSERT(invoked);
+      Q_ASSERT(QMetaObject::invokeMethod(rootObject(), "applyCurrentPaletteElement"));
```

Comment:
This will make `invokeMethod()` never be executed in release mode which is not what we want here.

Reply:
OK, so you're calling it for its side effects?